src/ring_theory/coprime/basic.lean

```diff
@@ -20,7 +17,7 @@ e.g., the multivariate polynomials `x₁` and `x₂` are not coprime.
 
 -/
```

Comment:
Could we add a "See also `ring_theory.coprime.lemmas` for further development of coprime elements."?